G/GDAL/build_tarballs.jl

```diff
@@ -91,6 +107,8 @@ dependencies = [
     Dependency("Libtiff_jll"; compat="4.3"),
     Dependency("libgeotiff_jll"; compat="100.700.100"),
     Dependency("LibCURL_jll"; compat="7.73"),
+    Dependency("NetCDF_jll"; compat="400.902.5"),
+    Dependency("HDF5_jll"),
```

Comment:
While you're changing the dependencies, it'd be better to change the version number, for example by bumping the version offset.